__tests__/components/MainDonationForm.spec.js

```diff
@@ -0,0 +1,65 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import MainDonationForm from '@components/MainDonationForm';
+
+const mockHandleSubmit = jest.fn();
+
+jest.mock('../../hooks/useStripeSession', () => ({
+  __esModule: true,
+  default: () => [mockHandleSubmit, false],
+}));
+
+describe('MainDonationForm', () => {
+  let portalRoot = document.getElementById('modal-root');
+  if (!portalRoot) {
+    portalRoot = document.createElement('div');
+    portalRoot.setAttribute('id', 'modal-root');
+    document.body.appendChild(portalRoot);
+  }
+
+  beforeEach(() => {
+    mockHandleSubmit.mockReset();
+    render(<MainDonationForm />);
+  });
+
+  it('should render switch, select and button', () => {
```

Comment:
```suggestion
  it('should render toggle switch, dropdown select combo box, and bottom button', () => {
```

Reply:
Looks like this test could use an `expect` to check rendering of the first item (the toggle switch). 🙂